### ISSUE TITLE
My jupyterlab extension only works with JupyterLab 1.0

### ISSUE THREAD

**Issue by mwouts** [jupyterlab/jupyterlab#6764]
Hello,

I have written an extension for JupyterLab, which was working well with JupyterLab 0.35 until recently, when it stopped working on that version, cf. https://github.com/mwouts/jupytext/issues/276.

Building JupyterLab 0.35 gives a warning like this one:
```
WARNING in @jupyterlab/apputils
  Multiple versions of @jupyterlab/apputils found:
    0.19.1 ./~/@jupyterlab\apputils from ./~/@jupyterlab\apputils\style\index.css
    1.0.0 ./~/jupyterlab-jupytext/~/@jupyterlab\apputils from ./~/jupyterlab-jupytext/~/@jupyterlab\apputils\lib\index.js
```
and then, when we open JupyterLab we don't see the extension. If we go to the browser logs, then we find this:
![image](https://user-images.githubusercontent.com/29915202/60689454-e33dda80-9ebd-11e9-87f0-8458cc310739.png)

Interestingly, the extension does work with JupyterLab 1.0. So maybe this is related to the recent release of JupyterLab ?

Thanks!

**Comment by jasongrout** [comment]
Yes, your npm semver range in your package.json is too broad and is picking up the 1.0 versions. you should probably be using ^ instead of >=

**Comment by mwouts** [comment]
I see! Thanks @jasongrout for the quick response. Now I know where to act! I'm curious to see if that means that I should prepare two separate versions, one for JLab beta, and the other one for JLab 1.0... I will give a try in the coming days. Thanks!

**Comment by jasongrout** [comment]
I think your jupytext extension may be on of the more popular ones (at least one that is getting more popular as word-of-mouth spreads news about it).

It probably makes sense to publish a patch version for 0.35 that fixes that semver range, so it keeps working, then update it for jlab 1.0.

**Comment by mwouts** [comment]
Thanks @jasongrout. I will do that soon.

**Comment by mwouts** [comment]
I have published two versions: 0.19 and 1.0 on npm, I think they will do the job. By the way, the only [difference between the two](https://github.com/mwouts/jupytext/commit/2dba8ab381422e18579cdb30d79bb491f7731a3c) is just the version of the dependencies - we agree that there is no way to detect the current version number on the fly, right? 

Again, thanks @jasongrout for your support on this.

**Comment by blink1073** [comment]
That looks right to me, @mwouts
